rootfs/opt/kubectl/example/GoogleBackup/manifest/statefulset-backup.tpl.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: <TIER>
+  labels:
+    tier: <TIER>
+    role: backup
+    app: qserv
+spec:
+  selector:
+    matchLabels:
+      tier: qserv
+      role: backup
+      app: qserv
+  serviceName: qserv
+  podManagementPolicy: "Parallel"
+  replicas: <WORKER_COUNT>
+  template:
+    metadata:
+      labels:
+        tier: qserv
+        role: backup
+        app: qserv
+    spec:
+      containers:
+        - name: sdk
+          image: "google/cloud-sdk:238.0.0"
+          imagePullPolicy: Always
+          command:
+            - <CMD>
+          volumeMounts:
+          - name: <TIER>-data
+            mountPath: /qserv/data
+          - name: secret-backup
+            mountPath: /secret-backup
+          - name: config-backup
+            mountPath: /config-backup
```

Comment:
Maybe it's just me, but seeing "config-backup" makes me think this is the backup for config, rather than a config for the backup system.  Probably nothing, just a bit confusing with the names.

Reply:
:+1:

Reply:
Please note that by convention, all Qserv configmaps start with "config-"